src/App.jsx

```diff
@@ -3,27 +3,64 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { loadTodos, loadUsers } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  async componentDidMount() {
+    const todosFromServer = await loadTodos();
+    const usersFromServer = await loadUsers();
```

Comment:
no need to fetch all users on mounting, you need to request one user in CurrentUser component